userland/libtock/led.c

```diff
@@ -1,18 +1,20 @@
-#include "led.h"
-#include "tock.h"
+#include "internal/led.h"
+
+// For LEDs, these simply pass through to the syscall wrappers (the functions
+// in the internal folder).
```

Comment:
So, timers basically have _two_ interfaces. One of them (the internal one) actually talks directly to the kernel timers. You would use this one if you wanted to bypass virtual timers, or implement your own, say to hook into a different language. The other actually implements virtual timers. The internal API is not a "supported API" because if you use it it'll screw up the virtual timers. So it's in internal cause you really shouldn't use it, but it's there.